src/templates/udmcfg.yaml.j2

```diff
@@ -0,0 +1,91 @@
+configuration:
+  keys:
+    udmProfileAHNPrivateKey: c53c22208b61860b06c62e5406a7b330c2b577aa5558981510d128247d38bd1d
+    udmProfileAHNPublicKey: 5a8d38864820197c3394b92613b20b91633cbd897119273bf8e4a6f4eec0a650
+    udmProfileBHNPrivateKey: F1AB1074477EBCC7F554EA1C5FC368B1616730155E0041AC447D6301975FECDA
+    udmProfileBHNPublicKey: 0472DA71976234CE833A6907425867B82E074D44EF907DFB4B3E21C1C2256EBCD15A7DED52FCBB097A4ED250E036C7B9C8C7004C4EEDC4F068CD7BF8D3F900E3B4
+  nrfUri: {{ nrf_url }}
+  sbi:
+    bindingIPv4: 0.0.0.0
+    port: {{ udm_sbi_port }}
+    registerIPv4: {{ udm_hostname }}
+    scheme: http
+    tls:
+      key: free5gc/support/TLS/udm.key
+      log: free5gc/udmsslkey.log
+      pem: free5gc/support/TLS/udm.pem
+  serviceNameList:
+  - nudm-sdm
+  - nudm-uecm
+  - nudm-ueau
+  - nudm-ee
+  - nudm-pp
+info:
+  description: UDM initial local configuration
+  version: 1.0.0
+logger:
+  AMF:
+    ReportCaller: false
+    debugLevel: info
+  AUSF:
+    ReportCaller: false
+    debugLevel: info
+  Aper:
+    ReportCaller: false
+    debugLevel: info
+  CommonConsumerTest:
+    ReportCaller: false
+    debugLevel: info
+  FSM:
+    ReportCaller: false
+    debugLevel: info
+  MongoDBLibrary:
+    ReportCaller: false
+    debugLevel: info
+  N3IWF:
+    ReportCaller: false
+    debugLevel: info
+  NAS:
+    ReportCaller: false
+    debugLevel: info
+  NGAP:
+    ReportCaller: false
+    debugLevel: info
+  NRF:
+    ReportCaller: false
+    debugLevel: info
+  NamfComm:
+    ReportCaller: false
+    debugLevel: info
+  NamfEventExposure:
+    ReportCaller: false
+    debugLevel: info
+  NsmfPDUSession:
+    ReportCaller: false
+    debugLevel: info
+  NudrDataRepository:
+    ReportCaller: false
+    debugLevel: info
+  OpenApi:
+    ReportCaller: false
+    debugLevel: info
+  PCF:
+    ReportCaller: false
+    debugLevel: info
+  PFCP:
+    ReportCaller: false
+    debugLevel: info
+  PathUtil:
+    ReportCaller: false
+    debugLevel: info
+  SMF:
+    ReportCaller: false
+    debugLevel: info
+  UDM:
+    ReportCaller: false
+    debugLevel: info
+  UDR:
+    ReportCaller: false
+    debugLevel: info
+  WEBUI:
+    ReportCaller: false
```

Comment:
I am not sure if we need this whole section.

Reply:
As far as I know, we kept it in the other operators, so I left it here for consistency. I can't remove it, I don't have a strong opinion.

Reply:
Changed my mind, removed.

Reply:
We've been removing those in some charms, look at the [NSSF config file](https://github.com/canonical/sdcore-nssf-operator/blob/main/src/templates/nssfcfg.conf.j2) Do you we have any proof we need it? Probably the same as the comment below, we should agree to follow the same pattern.

Reply:
No, I don't have any proof that we need this. I also don't have any proof we don't :) 
That's why I'm asking.

Reply:
Let's add it to the discussion on the daily.

Reply:
I've added it back as discussed in the daily.